### ISSUE TITLE
About validation and test

### ISSUE THREAD

**Issue by ENDLICHERE** [dxli94/WLASL#19]
I have 3 questions about the validation and test in I3D. (I use the subset WLASL100) Hope someone could help me.
1.  In ./I3D/train.py line 57, it seems that it is the 'testing' sets rather than 'validation' sets are used for validation process. Thus I wonder that it is a mistake or just the special setting of this work.
2.  I utilize the given weight './archived/asl100/FINAL_nslt_100_iters=896_top1=65.89_top5=84.11_top10=89.92.pt' to test the I3D model, and get the result: top1=67.07, top5=84.58, top10=90.25, which is different from the accuracy presented in the name of the weight. Why would this happen and which one should be chose as the final result？
3.  I once trained the I3D model, using 'tesing' set for  both validation and testing. But the validation results and the testing results are different, which makes me confused.

**Comment by CHI-YU-SUNG** [comment]
I have the same problem for the first two questions

**Comment by dxli94** [comment]
(1) As we reported in the paper, we use the validation set to select the hyperparameter. Then we merge the validation set and training set for training and report the result on the testing set with the selected hyperparameters. This is a common practice when the training set is far from being sufficient for training a good model.

(2) At this stage I cannot comment on such a minor difference. Please refer to the original paper for comparisons.

(3) During training, only a fixed number of video frames are sampled for the ease of batching. However, during testing, all of the video frames are fed to the recognition model. This leads to a small gap in the performance difference.

**Comment by ENDLICHERE** [comment]
Thank you very much for your prompt reply.
